example/libhdbpp-example/CMakeLists.txt

```diff
@@ -0,0 +1,241 @@
+# Functions and Pre-build  -----------------------------------
+
+# Stop messy in source builds
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+set(CMAKE_DISABLE_SOURCE_CHANGES  ON)
+
+if ( ${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR} )
+    message( FATAL_ERROR "In-source builds not allowed. Please make a new directory (called a build directory) and run CMake from there. You may need to remove CMakeCache.txt." )
+endif()
+
+# Start Build Config -----------------------------------
+cmake_minimum_required(VERSION 3.11)
+set(CMAKE_SKIP_RPATH true)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_COLOR_MAKEFILE ON)
+
+# Library output name for the final binaries
+set(LIBHDBPP_EXAMPLE_NAME "libhdb++example")
+
+# Versioning
+set(VERSION_MAJOR "0")
+set(VERSION_MINOR "0")
+set(VERSION_PATCH "0")
+set(VERSION_METADATA "")
+set(VERSION_STRING ${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH})
+
+# Add any include paths from the command line
+list(APPEND INCLUDE_PATHS ${CMAKE_INCLUDE_PATH})
+list(APPEND INCLUDE_PATHS ${CMAKE_SOURCE_DIR})
+list(APPEND LIBRARY_PATHS ${CMAKE_LIBRARY_PATH})
+
+# Start the project
+project(libhdbpp_example VERSION ${VERSION_STRING} LANGUAGES CXX)
+
+# Require this for the libpqxx build, also this project is c++14 minimum
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# Build options
+option(BUILD_UNIT_TESTS "Build unit tests" OFF)
+option(ENABLE_CLANG "Enable clang code and layout analysis" OFF)
+
+if(BUILD_UNIT_TESTS)
+    message(STATUS "Unit tests will be built")
+endif(BUILD_UNIT_TESTS)
+
+# arch install definitions
+include(GNUInstallDirs)
+
+message(STATUS "Searching for libraries...")
+
+# allow pkg-config to search the CMAKE_PREFIX_PATH 
+set(PKG_CONFIG_USE_CMAKE_PREFIX_PATH ON)
+list(APPEND CMAKE_PREFIX_PATH "/usr")
+
+# Find Dependencies  ---------------------
+include(cmake/FindLibraries.cmake)
+
+# First find tango if it has not already been found. Returns an interface library
+# called TangoInterfaceLibrary
+set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+find_package(Tango)
+
+# Find the libhdbpp headers
+find_package(libhdbpp)
+
+# If not installed then get them
+if(NOT libhdbpp_FOUND)
+    include(FetchContent)
+
+    FetchContent_Declare(
+      libhdbpp
+      GIT_REPOSITORY https://github.com/tango-controls-hdbpp/libhdbpp.git
+      GIT_TAG        project-build
```

Comment:
GIT_TAG should be master since we're merging

Reply:
Wasn't sure about this one indeed, as for the current template to work we need project-build. But let's go for master directly !

Reply:
It's a committment to merge soon all pending PR.